qualtran/bloqs/block_encoding/lcu_block_encoding.py

```diff
@@ -176,7 +176,7 @@ def get_single_qubit_controlled_bloq(self, control_val: int) -> 'LCUBlockEncodin
 
 
 @attrs.frozen
-class LCUBlockEncodingZeroState(BlockEncoding, SpecializedSingleQubitControlledExtension):
+class LCUBlockEncoding(BlockEncoding, SpecializedSingleQubitControlledExtension):
     r"""LCU based block encoding using SELECT and PREPARE oracles.
```

Comment:
add a line explaining how this differs from `SelectBlockEncoding`?

Reply:
Also the docstring below should use signal state 0 instead of G. (lines 197-199)

Reply:
Ah I misread the `I_a`, yeah this is fine. But still I feel it is better to just avoid G altogether and just say <0|B[H]|0>, makes it simpler to read. But either is fine.

Reply:
I will defer to you.